bundler/lib/bundler/source_list.rb

```diff
@@ -88,7 +88,7 @@ def get(source)
     def lock_sources
       lock_sources = (path_sources + git_sources + plugin_sources).sort_by(&:to_s)
       if disable_multisource?
-        lock_sources + rubygems_sources.sort_by(&:to_s)
+        lock_sources + rubygems_sources.sort_by(&:to_s).uniq
```

Comment:
I'm just wondering if duplication in `rubygems_sources` would not cause similar problem in any other place (like in `all_sources`). :thinking:

Reply:
I wondered the same thing and tried changing `rubygems_sources` too, but changing that method broke some other specs, so I cancelled that.